apps/server/src/modules/tool/service/tool-validation.service.spec.ts

```diff
@@ -278,6 +302,16 @@ describe('ToolValidation', () => {
 				await expect(result).resolves.not.toThrow();
 			});
 
+			it('should validate the default with the regular expression', async () => {
```

Comment:
this is not testing what it says. If the default was not validated at all, the test would also pass.

Reply:
I would guess that you probably did not check if this test fails without the code you added. A test that doesnt fail when its condition is not met is very, very dangerous, its worse than no test at all.

In the future please pay attention to this, I as a reviewer cant possibly always notice it.

(it will also help with not duplicating tests, because its slightly more difficult to make a test fail that already exists in a different place :P)

Reply:
... this is actually code that should have been removed for the describes below ^^ good eye! :)